nixos/modules/services/networking/murmur.nix

```diff
@@ -251,19 +273,13 @@ in
       after       = [ "network-online.target "];
 
       serviceConfig = {
-        Type      = "forking";
+        Type      = "simple";
         RuntimeDirectory = "murmur";
-        PIDFile   = cfg.pidfile;
         Restart   = "always";
         User      = "murmur";
-        ExecStart = "${pkgs.murmur}/bin/murmurd -ini ${configFile}";
+        ExecStart = "${cli-wrapper}/bin/murmurd-service -fg";
         PermissionsStartOnly = true;
```

Comment:
I know https://github.com/NixOS/nixpkgs/pull/56265 isn't related to this PR at all... but would you be willing to do me a solid and remove `PermissionsStartOnly = true;` in an extra commit here? It looks like the only other things to do would be to add `StateDirectory = "murmur";` and remove the call to `createHome = true;` and you would be good to go...